alinka/widget/containers/main_body/footer/browser.py

```diff
@@ -0,0 +1,31 @@
+from PySide6.QtWidgets import QFrame, QHBoxLayout, QPushButton, QWidget
+
+from alinka.db.queries import get_decision_data_by_id
+from alinka.docx.utils import convert_raw_documents_data
+
+
+class BrowserFooterContainer(QFrame):
+    def __init__(self, parent: QWidget, visible: bool = False):
+        super().__init__(parent)
+        self.footer_container = parent
+        layout = QHBoxLayout(self)
+        layout.setContentsMargins(9, 9, 9, 9)
+        self.create_new_btn = QPushButton("Utwórz nowy", self)
+        self.create_new_btn.clicked.connect(self.create_new_decision)
+        self.create_new_btn.setEnabled(False)
+        layout.addWidget(self.create_new_btn)
+
+        if not visible:
+            self.setVisible(visible)
```

Comment:
What's the purpose of that if-statement? Cannot we just
```suggestion
        self.setVisible(visible)
```

Reply:
Indeed! Thanks!